.github/CONTRIBUTING.rst

```diff
@@ -0,0 +1,53 @@
+Thank you for your interest in contributing to Junction. We welcome all
+contributions and greatly appreciate your effort!
+
+Bugs and Features
+-----------------
+
+If you have found any bugs or would like to request a new feature, please do
+check if there is an existing issue already filed for the same, in the
+project's GitHub `issue tracker`_. If not, please file a new issue.
+
+If you want to help out by fixing bugs, choose an issue from the `issue
+tracker`_ to work on and claim it by posting a comment saying "I would like to
+work on this.". Feel free to ask any doubts in the issue thread.
+
+Once you have implemented the feature to an extent, go ahead and file a pull
+request, following the tips below. File a pull request early to get feedback as
+early as possible.
+
+Pull Requests
+-------------
+
+Pull Requests should be small to facilitate easier review. Keep them
+self-contained, and limited in scope. Studies have shown that review quality
+falls off as patch size grows. Sometimes this will result in many small PRs to
+land a single large feature.
+
+Checklist:
```

Comment:
Should we also mention that `you should always create a new branch for a new pr to avoid conflicts`?